packages/cms-lib/lib/constants.js

```diff
@@ -87,6 +87,18 @@ const ENVIRONMENT_VARIABLES = {
   HUBSPOT_REFRESH_TOKEN: 'HUBSPOT_REFRESH_TOKEN',
 };
 
+const SCOPE_GROUPS = {
+  functions: 'cms.functions.read_write',
+};
+
+const SCOPES = {
+  functions: {
+    access: 'SERVERLESS_FUNCTIONS',
+    read: 'SERVERLESS_FUNCTIONS_READ',
+    write: 'SERVERLESS_FUNCTIONS_WRITE',
```

Comment:
I am concerned that this will fall out of date when/if scopes or scope groups change.

Reply:
I removed the `SCOPES` constants. We still need the `SCOPE_GROUPS.functions` scope group in order to hit the endpoint properly.